rpcs3/Emu/RSX/VK/VKDraw.cpp

```diff
@@ -161,11 +161,12 @@ void VKGSRender::load_texture_env()
 			(update_framebuffer_sourced && fs_sampler_state[i]->upload_context == rsx::texture_upload_context::framebuffer_storage))
 		{
 			auto sampler_state = static_cast<vk::texture_cache::sampled_image_descriptor*>(fs_sampler_state[i].get());
+			auto texture = rsx::method_registers.fragment_textures[i];
```

Comment:
`auto& tex` for consistency with the rest of the codebase

Reply:
Fix it.
https://github.com/RPCS3/rpcs3/pull/9346/commits/f7c2dcd6ac1e185d2745ab681e12225f8ea2f9d5